Do not build input_raspicam plugin which depends on
depreciated MMAL APIs.
Default V4L2 APIs are now supported on Pi

---
 mjpg-streamer-experimental/CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index cf26620e73b..38ff8e31c39 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -59,7 +59,6 @@ find_library(JPEG_LIB jpeg)
 add_subdirectory(plugins/input_file)
 add_subdirectory(plugins/input_http)
 add_subdirectory(plugins/input_opencv)
-add_subdirectory(plugins/input_raspicam)
 add_subdirectory(plugins/input_ptp2)
 add_subdirectory(plugins/input_uvc)
 
-- 
2.53.0.dirty

